Kiosk/Bid Fulfillment/LoadingViewController.swift

```diff
@@ -19,11 +19,16 @@ class LoadingViewController: UIViewController {
     @IBOutlet weak var backToAuctionButton: SecondaryActionButton!
     @IBOutlet weak var placeHigherBidButton: ActionButton!
 
+    private let _viewWillDisappear = PublishSubject<Void>()
+    var viewWillDisappear: Observable<Void> {
```

Comment:
I get this pattern now, you're hiding the fact that it's actually a Observable that this class can trigger.